src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: ../../layouts/ContentLayout.astro
+---
+
+# Ce qui caractérise le top 20% des devs
+
+## Introduction
+
+Lorsque l'on parle des développeurs, il est fréquent de vouloir comprendre ce qui distingue les meilleurs, le fameux top 20%, du reste des développeurs.
+
+Que font-ils différemment ?
+
+Quels sont leurs traits de caractère, leurs habitudes ou leurs compétences spécifiques qui les propulsent au sommet de leur domaine ?
+
+Je suis Oumar, j’ai rejoins le monde de la programmation en septembre 2022 et c’est là tout un tas de question que je me suis posé (et que je me pose toujours d’ailleurs pour sainement progresser 😉).
+
+Je vous propose donc dans ce blog post de vous partager les conseils reçus, de mes paires plus “seniors”. Cela me sers beaucoup au quotidien et je l’espère vous servira également. Allez on se lance !
+
+## Oui, la technique ça compte
+
+### La curiosité et la résilience
+
+Aussi technique soi le sujet cela ne les refrènent pas le moins du monde, c’est un nouvel obstacle parmi tant d’autres à surmonter. Cela est même souvent source de plaisir pour beaucoup.
+
+Notre métier est sacrément dur faut le dire et le reconnaître !
+
+En effet, les meilleurs développeurs sont souvent des personnes curieuses-résilientes (j’insiste sur la mise en commun de ces deux caractéristiques) qui s'intéressent à de nouvelles technologies, outils ou langages de programmation. Ils aiment expérimenter avec de nouvelles idées et ne se contentent pas de suivre les méthodes établies. Ils cherchent toujours à comprendre comment les choses fonctionnent en profondeur. C’est souvent de la que ressort l’un des deux groupes du top 20%, que j’entends souvent qualifier “d’innovateurs”. On parlera du second groupe, dans la prochaine caractéristique.
+
+### Approche orientée problème à la solution (et non l’inverse)
+
+Ils ont une approche orientée problème, ce qui signifie qu'ils se concentrent sur la résolution de problèmes spécifiques plutôt que sur l'application d'une technologie en elle-même.
+
+Ils choisissent les outils et les méthodes en fonction de ce qui est le mieux adapté pour résoudre le problème à portée de main. C’’est souvent de là que ressort ceux que j’appel (excusez mon anglicisme), les “problèmes solveurs” qui à force d’entrainer cette capacité de résolution de problème développent une vélocité cumulée à une efficacité qui en fait des développeurs hors norme.
+
+C’est le gars dans l’équipe (pas forcément le plus senior) à qui on pense dont on est quasi sûr qu’il résoudra aussi vite et proprement un problème qui nous aura pris un temps fou.
+
+### L’ Apprentissage Continu
+
+Etant un profil en reconversion venant de la finance, un élément m’a frappé dans ce nouveau monde fascinant. Tout le monde est à la recherche du One Piece (clin d’œil aux fan de mangas) et cela change la donne par rapport à la plupart des autres industries (dont la finance).
```

Comment:
It's great to quote a reference. However, you should example the term "One Piece" for people, like me, who don't know what it means.

Reply:
Done. 🙂